pkg/server/container_create_unix.go

```diff
@@ -193,9 +193,9 @@ func (c *criService) containerSpec(id string, sandboxID string, sandboxPid uint3
 		if !sandboxConfig.GetLinux().GetSecurityContext().GetPrivileged() {
 			return nil, errors.New("no privileged container allowed in sandbox")
 		}
-		specOpts = append(specOpts, oci.WithPrivileged)
+		specOpts = append(specOpts, oci.WithPrivileged, oci.WithAllDevicesAllowed)
 		if !ociRuntime.PrivilegedWithoutHostDevices {
```

Comment:
let's consider adding a new flag PrivilegedWithoutAllDevicesAllowed default false...

Reply:
I'm happy to add a new flag, my only concern is what behaviour do we want when both `PrivilegedWithoutHostDevices` and the new flag are true? i.e. what happens to `oci.WithAllDevicesAllowed`?

Reply:
document that you can't have both.. and we can check for one or the other or neither in the config validate code..